src/test/debuginfo/thread-names.rs

```diff
@@ -0,0 +1,40 @@
+// compile-flags:-g
+// We can't set the main thread name on Linux because it renames the process (#97191)
+// ignore-linux
```

Comment:
Oh hm, we might need to be a bit more wide ranging with the excluded OSes. Maybe something like:

```suggestion
// ignore-linux
// ignore-android
// ignore-dragonfly
// ignore-emscripten
// ignore-freebsd
// ignore-haiku
// ignore-ios
// ignore-netbsd
// ignore-openbsd
// ignore-solaris
// ignore-sgx
```

Reply:
Thanks!